src/App.scss

```diff
@@ -239,6 +239,52 @@ body {
     justify-content: center;
 }
 
+
+.App__ShareButton {
+    background-color: #33717F;
+    border: 0;
+    border-radius: 0.2rem;
+    color: #FFFFFF;
+    font-weight: 400;
+    padding: 0.5rem 2rem;
+    width: 10rem;
+
+    display:flex;
+    grid-template-columns: 30% 70%;
```

Comment:
This grid-template is not applying, because display is set to flex, but I think we don't need this template at all, since flex layout is handling it nicely. Let's remove this and remove white space on line 251 as well.

Reply:
Done.